Utils/Dataflow/071_esConsistency/README

```diff
@@ -28,4 +28,3 @@ The stage can be run as following:
 For more information about running the stage and its arguments, use:
 
   ./consistency.py -h
-
```

Comment:
The commit log states: `Remove second trailing empty line.`
Why is it "second"?

Reply:
Changed "second" to "extra".